src/Card/README.md

```diff
@@ -526,7 +602,8 @@ behavior.
     </Card.Footer>
   </Card>
   <Card>
-    <Card.ImageCap 
+    <Card.ImageCap
+      skeletonHeight={140}
```

Comment:
If this `Card` isn't in a loading state (no `isLoading` prop), do we need the `skeletonHeight` prop here?